src/python_framework/nhd_network_traversal.py

```diff
@@ -2,34 +2,60 @@
 A demonstration version of this code is stored in this Colaboratory notebook:
     https://colab.research.google.com/drive/1ocgg1JiOGBUl3jfSUPCEVnW5WNaqLKCD
 
+
+## command line input order [debuglevel = 0,verbose = true,showtiming = true, supernetworks = supernetwork_options]
 """
 import nhd_network_utilities as nnu
 import recursive_print
 import time
 import os
+import sys
 
-# NOTE: these methods can lose the "connections" and "rows" arguments when
-# implemented as class methods where those arrays are members of the class.
-
+try:
```

Comment:
https://docs.python.org/3/library/argparse.html

I would reccommend using argparse, it is a great module that takes care of a lot of the backend of reading sys.argv for command line args.  For a quick example, see https://github.com/NOAA-OWP/DMOD/blob/5fbc094c9de950516432a594c2932018fdc6be67/python/services/schedulerservice/dmod/schedulerservice/__main__.py
Once the parser is built, you simply parse with `args = parser.parse_args()` and they are all available as `args.arg_name`.